CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
```

Comment:
note that the braces are meant to turn this into a link. if you check [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) they have the links to the commit history at the end (in markdown there are several ways to define links, you don't need to have the actual URL in the text - in the rendered text you won't notice the difference).

Reply:
Hmm, I copied from their markdown that does render those links nicely.  I wonder if GitHub does something special when rendering the markdown on the website - I'll look into this.

Reply:
you're missing this part:

https://github.com/olivierlacan/keep-a-changelog/blob/ee8cc0c6256716bb57af41f7e58001fb58ae193d/CHANGELOG.md?plain=1#L193-L206

Reply:
Oh! That so fancy